tokens/governance/treasury-spend.md

```diff
@@ -0,0 +1,93 @@
+    
+=== "Moonbase Alpha"
+    |         Variable         |                                                          Value                                                          |
+    |:------------------------:|:-----------------------------------------------------------------------------------------------------------------------:|
+    |      Proposal Bond       | {{ networks.moonbase.treasury.proposal_bond}}%  |
+    |     Minimum Bond     |   {{ networks.moonbase.treasury.proposal_bond_min}} DEV |
+    |     Spend Period     |   {{ networks.moonbase.treasury.spend_period_days}} days |
+
+
+This guide will show you how to submit a proposal on Moonbase Alpha. It can be adapted for Moonbeam or Moonriver. In any case, it's recommended that you familiarize yourself with the steps of submitting a treasury proposal on Moonbase Alpha or a local dev node before taking the steps on Moonbeam or Moonriver. 
+
+## Roadmap of a Treasury Proposal {: #roadmap-of-a-treasury-proposal } 
+
+You can find a full explanation of the [happy path for a treasury proposal here](/learn/features/treasury/){target=_blank}. For more information on the process, see this [guide to the interim community treasury program](https://github.com/moonbeam-foundation/treasury/blob/main/interim/interim_treasury_proposal.md){target=_blank}.
+
+![Proposal Roadmap](/images/tokens/governance/treasury-proposals/treasury-proposal-roadmap.png)
+
+## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+
+It's highly recommended that you preface any proposal with a post on [Moonbeam's Community Forum](https://forum.moonbeam.foundation/){target=_blank}. This especially important for treasury proposals because of the irrevocable bond required to submit a treasury spend proposal. You can submit your idea for a treasury spend proposal by following [these instructions](https://moonbeam.network/blog/using-moonbeam-community-forum/){target=_blank}. 
+
+![Moonbeam Forum Home](/images/tokens/governance/treasury-proposals/treasury-proposal-1.png)
+
+## Proposing an Action {: #proposing-an-action } 
+
+This guide focuses on the mechanics of submitting an on-chain treasury proposal after you have worked with the Treasury Council to refine your idea. If you haven't yet completed the prior steps of the treasury proposal process, please take a moment to review the [guidelines of the community treasury program](https://github.com/moonbeam-foundation/treasury/blob/main/interim/interim_treasury_proposal.md){target=_blank} and evaluate your standing. Engaging the community and revising your proposal based on feedback received is the most critical piece of your proposal. Submitting the on-chain treasury proposal is the easy part, demonstrated in the following guide. 
+
+To get started, head to the **Governance** section and click on **Treasury**. Then click **Submit Proposal**.
+
+![Treasury home](/images/tokens/governance/treasury-proposals/treasury-proposal-2.png)
```

Comment:
nitpick, sorry haha but can you crop out the page address at the bottom of the image